docs/Reference/API-Methods.md

```diff
@@ -74,7 +74,7 @@ You can specify only one log level per RPC call.
 
 #### Returns
 
-`result` : `Success` if the log level has changed; otherwise `error`.
+`result` : `Success` if the log level has changed, otherwise `error`.
```

Comment:
there is still some inconsistency as to whether "otherwise" should be followed by a "," or not